docs/housing_search.md

```diff
@@ -24,6 +24,15 @@ ElasticSearch will be updated by a Lambda function, holding business logic, that
 ** Search by : **
 1. Person - First name, Middle name, Last name
 2. Asset - Address line 1, Postcode, Asset type
+3. Tenure - Payment Reference, FullAddress of TenuredAsset , Household Members FullName  
+4. Transactions - Person FullName, Transaction Type, Payment Reference, Bank Account Number, Transaction Date, Transaction Amount  
```

Comment:
Can we make this "Sender name" instead of "Person Full Name" to make it applicable to both a person's name and a company's name?

Reply:
Will change Person Full Name to Sender name